librz/bin/format/elf/elf_dynamic.h

```diff
@@ -0,0 +1,16 @@
+// SPDX-FileCopyrightText: 2021 08A <08A@riseup.net>
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-License-Identifier: LPGL-3.0-only
+
+#include <ht_uu.h>
+#include "elf.h"
```

Comment:
Swapping these fixes the `eprintf()` warning (the sdb one contains an explicit `#undef` so it works if that comes later)